packages/trader/src/Modules/CFD/Containers/cfd-password-manager-modal.tsx

```diff
@@ -179,7 +195,7 @@ const InvestorPasswordManager = ({
                 {({ isSubmitting, errors, setFieldTouched, values, touched }) => (
                     <Form className='cfd-password-manager__investor-form' noValidate>
                         <Field name='old_password'>
-                            {({ field }) => (
+                            {({ field }: FieldProps) => (
```

Comment:
As per Formik docs, the correct usage of FieldProps here would be the following:
```suggestion
                            {({ field }: FieldProps<string, TFormValues>) => (
```
`TFormValues` can be declared at the beginning of this file as a type of all form values collected in this particular form specifically like this:
```
type TFormValues = {
    old_password: string;
    new_password: string;
    password_type: string;
};
```
I suggest also using `TFormValues` type at line 287 as type of `values` in validatePassword parameters.